README.md

```diff
@@ -297,6 +297,7 @@ Sometimes you have tests that work well for some platform combinations, e.g. Apa
 
 Tests can be altered using four lists:
 - `input` allows you to override global parameters in tests. An example usage is if you want to change the `dest_addr` of all tests to point to an external IP or host
+  - it includes also the `override_empty_host_header` flag: enabling it, empty Host headers will be replaced with `dest_addr`
```

Comment:
I think we should explicitly document all available overrides. That will make this section clearer.

Reply:
Done, I tried to write a small description of each, but not sure about `savecookie` and `stopmagic` 🤔

Reply:
Thanks a lot!